compiler/src/iree/compiler/Dialect/Flow/Conversion/TensorToFlow/Patterns.cpp

```diff
@@ -183,6 +183,55 @@ struct ConvertTensorFromElementsPattern
   }
 };
 
+/// Populates given sizes array from type (for static sizes) and from
+/// the tensor (for dynamic sizes).
+static void sizesForTensor(OpBuilder &builder, SmallVectorImpl<Value> &sizes,
+                           Location loc, ShapedType stp, Value tensor) {
+  for (const auto &d : enumerate(stp.getShape())) {
+    Value dim;
+    if (d.value() == ShapedType::kDynamic)
```

Comment:
Nit: IREE style is to add `{` `}` everywhere.

Reply:
done